### ISSUE TITLE
directory assumes parent folder as root.

### ISSUE THREAD

**Issue by tinstep** [jrnewell/spotify-ripper#60]
If I put in my config.ini file 

directory = /share/music

it assumes the folder is in the user folder

/home/<user>/share/music

i think if the path starts with a "/" it should assume it means root.

trying to append ../../to get back to root didnt work either.
my config.ini resides in /home/<user>/.spotify-ripper

**Comment by jrnewell** [comment]
I am having difficulty reproducing your issue.  For example, I tried this and got a correct absolute path output.  Can you send me your config.ini and a log with sensitive info removed?

config.ini
```INI
[main]
last = True
overwrite = True
directory = /Users/james/Downloads
format = {album_artist}/({year}) - {album}/{track_num} - {track_name}.{ext}
```

Log
```
Spotify Ripper - v2.1.1
  Encoding output:	MP3, VBR 0
  Spotify bitrate:	320 kbps
  Unicode support:	Yes
  Output directory:	/Users/james/Downloads
  Settings directory:	/Users/james/.spotify-ripper
  Format String:	{album_artist}/({year}) - {album}/{track_num} - {track_name}.{ext}
  Overwrite files:	Yes
Logging in...
Logged in as jrnewell
Loading starred playlist...
Total Download Size: 17.28 MB
Loading track...
[ 1 / 2 ] Ripping spotify:track:5X19PEYSahBJnyQZVUVPG5
/Users/james/Downloads/Mind.in.a.box/(2010) - R.E.T.R.O/6 - 8 Bits.mp3
Track Download Size: 7.93 MB
Progress: [===============                         ] 01:22 / 03:28        (~01m 25s remaining)
Total:    [=======                                 ] 01:22 / 07:33        (~04m 00s remaining)^C
Aborting...
Deleting partially ripped file
Logging out...
```

**Comment by tinstep** [comment]
As soon as I try a path with a space like
directory = "/share/Holding/downloads/music for processing/_spotify_rips/"
I get the error,  I have changed the path so i dont have any folders with spaces to test - but now i get the "spotify error detected" (which is already listed as  bug.

**Comment by tinstep** [comment]
I can confirm that when using a path without quotes the directory is selected correctly, when encapsulating a path in " it errors.


Spotify Ripper - v2.1.2
  Encoding output:	FLAC, Compression Level: 8
  Spotify bitrate:	320 kbps
  Unicode support:	Yes
  Output directory:	/share/Holding/downloads/_spotify_rips    <<<<<< this is correct!!!
  Settings directory:	/home/cam/.spotify-ripper
 

Here is the output of the old Bad error:
  Output directory:	/home/user/share/Holding/downloads/music for processing/_spotify_rips/"  <<<<Not correct

(note there is one double quote at the end.)
the config file held:
directory = "/share/Holding/downloads/music for processing/_spotify_rips/"


hope this helps.

**Comment by jrnewell** [comment]
Yeah it looks like python's configparser isn't smart enough to remove quotes around strings.  I will add some code to strip quotes to prevent this issue.

**Comment by jrnewell** [comment]
Fixed in version ``2.1.3``.
